{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -190,12 +190,23 @@
 # Custom user model
 AUTH_USER_MODEL = "users.User"
 
-EMAIL_HOST = env.str("EMAIL_HOST", "smtp.sendgrid.net")
-EMAIL_HOST_USER = env.str("SENDGRID_USERNAME", "")
-EMAIL_HOST_PASSWORD = env.str("SENDGRID_PASSWORD", "")
+
+# Configuration sendgrid api start
+EMAIL_BACKEND = 'sendgrid_backend.SendgridBackend'
+
+DEFAULT_FROM_EMAIL = 'Crowdbotics <noreply@crowdbotics.com>'
```

Comment:
This should be `support@crowdbotics.com`. Nonexistent email addresses are a bad practice.

Reply:
I am just using demo purpose you can change according to yourself, whatever you need.